docs/iris_archive.md

```diff
@@ -28,6 +28,39 @@ To easily browse the IRIS Archive with a graphical interface, Azure Storage Expl
 
 For each dataset Azure Storage Explorer caches 5000 files, and only shows these by default. To load additional files the `Load more` button must be pressed.
 
+
+### Tag filtering
+
+All of the blobs inside the IRIS Archive have 3 tags that can be used for filtering:
+ - Dataset - The acronym for the dataset name (e.g. AGPT)
+ - Timestamp - Reference time for the blob's data content
+ - UploadTime - The time when the blob was uploaded to IRIS
+
+More information about dataset acronyms and reference timestamp fields can be found [here](dataset_fields_for_iris_timestamp.md).
+
```

Comment:
I think it would be good to flag 2 additional things: 
1 - format of the datetimes used in the tags
2 - tags are case sensitive